src/gtc/gtcpp/oir_to_gtcpp.py

```diff
@@ -194,7 +194,7 @@ def visit_VerticalLoop(
             comp_ctx=comp_ctx,
             **kwargs,
         )
-        caches: List[Union[IJCache]] = []  # TODO
+        caches: List[Union[IJCache]] = []  # TODO(havogt): caches are not implemented
```

Comment:
`Union[IJCache]` is equivalent to `IJCache`, did you mean `Optional[IJCache]`?

Reply:
Actually this was preparing for `List[Union[IJCache, KCache]]`. For this type annotation, I just copied from https://github.com/havogt/gt4py/blob/b7f8a50a6b011dc2d071307dd27f69a6bb20a4c8/src/gtc/gtcpp/gtcpp.py#L240. I'll leave it. This will be fixed in the next cycle.